src/packets.c

```diff
@@ -798,7 +798,7 @@ TbBool process_dungeon_control_packet_dungeon_control(long plyr_idx)
               } else
               if (dungeon->task_count < 300)
               {
-                if (can_dig_here(stl_x, stl_y, player->id_number))
+                if (can_dig_here(stl_x, stl_y, player->id_number, false))
```

Comment:
I am not sure of the impact of setting enemy_wall_undiggable to true here.

Unless anyone else can work out the impact, I suggest leaving this as false too.